### ISSUE TITLE
Misleading error message when no input files are detected

### ISSUE THREAD

**Issue by arvigeus** [DavidAnson/markdownlint#304]
```sh
markdownlint --fix '**/*.md'
```

When running this command in a project without any md files, it gives me confusing error:

```sh
  Usage: markdownlint [options] <files|directories|globs>

  MarkdownLint Command Line Interface

  Options:

    -h, --help                                  output usage information
    -V, --version                               output the version number
    -f, --fix                                   fix basic errors (does not work with STDIN)
    -s, --stdin                                 read from STDIN (does not work with files)
    -o, --output [outputFile]                   write issues to file (no console)
    -c, --config [configFile]                   configuration file (JSON, JSONC, JS, or YAML)
    -i, --ignore [file|directory|glob]          file(s) to ignore/exclude
    -p, --ignore-path [file]                    path to file with ignore pattern(s)
    -r, --rules  [file|directory|glob|package]  custom rule files
```

It was my mistake to forget to add a readme file, but I think in this case the output should be something more specific, like `No valid files found`. Not it suggest I entered invalid command.

**Comment by DavidAnson** [comment]
Agreed. Please open this issue in the CLI project because I can’t transfer it: https://github.com/igorshubovych/markdownlint-cli

**Comment by arvigeus** [comment]
Closed in favor of [markdownlint-cli#104](https://github.com/igorshubovych/markdownlint-cli/issues/104)
